index.html

```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
             </nav>
         </header>
         <main>
-            <section>
+            <div class="main">
+                <section id="about-me">
+                    <div>
+                        <div>
+                            <h1>
+                                I'm William
+                                Glad to see you!
+                            </h1>
+                        </div>
+                        <div class="description">
+                            <p>I'm a software developer! I can help you build a product , feature or website Look
+                                through
+                                some
+                                of my
+                                work and
+                                experience! If you like what you see and have a project you need coded, don’t hestiate
+                                to
+                                contact
+                                me.</p>
+                        </div>
+                        <div class="connect">
+                            <h5>LET'S CONNECT</h5>
```

Comment:
I think you should kindly wrap this in a `<h2>` if you have to use an `h` tag. it is important you always note that an h2 falls under an h1, an h3 is a heading under an h2, etc always remember the hierarchy, this is semantically and aids user accessibility. 🤗